corebuild/integration/ILLink.Tasks/ILLink.Tasks.targets

```diff
@@ -383,6 +401,7 @@
       <_LinkerRootAssemblies Include="@(_ManagedResolvedAssembliesToPublish->'%(Filename)')" />
       <_LinkerRootAssemblies Remove="@(PlatformLibraries->'%(Filename)')" />
       <_LinkerRootAssemblies Remove="System.Private.CoreLib" Condition=" '$(_SPCHasEmbeddedRootDescriptor)' == 'true' "/>
+      <_LinkerRootAssemblies Remove="@(_DuplicateManagedAssembliesToSkip->'%(Filename)')" />
```

Comment:
Will this keep the duplicate representative assemblies that we will link as roots?

Reply:
It will if they're not part of the platform.

Reply:
You're right - this was actually removing the representative assemblies completely. Fixed in latest commit.